samcli/lib/list/mapper_consumer_factory_interface.py

```diff
@@ -0,0 +1,10 @@
+"""
+Interface for MapperConsumerFactory
+"""
+import abc
+
+
+class MapperConsumerFactoryInterface:
```

Comment:
I believe that abstract classes in Python should inherit from `ABC`.

Reply:
Added the inheritance